common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -54,7 +54,6 @@
         border-bottom: 0.0625rem dotted #dfdfdf;
```

Comment:
why have all the file modes changed from 644 to 755?  I think this is wrong, they should be executable